clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -200,12 +200,15 @@ func (e *etcdMetricsMeasurement) getEtcdMetrics(host string, provider provider.P
 
 	// Use old endpoint if new one fails, "2379" is hard-coded here as well, it is kept as is since
 	// we don't want to bloat the cluster config only for a fall-back attempt.
-	etcdCert, etcdKey := os.Getenv("ETCD_CERTIFICATE"), os.Getenv("ETCD_KEY")
+	etcdCert, etcdKey, etcdHost := os.Getenv("ETCD_CERTIFICATE"), os.Getenv("ETCD_KEY"), os.Getenv("ETCD_HOST")
```

Comment:
I'd love to have these as "FlagEnvs" (which are properly documented) rather than just env variables, but given it's written this way already I won't be blocking it. 

If you have a time to migrate these env variables to flags then I'd really appreciate :)